test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -196,15 +216,30 @@ export default class PerfCompare extends Component {
         let resBenchmarkBaselineJson = await baselineBuildTestInfo.json();
```

Comment:
Maybe we should rename these variables since it's slightly confusing. We could call it `baselineBuildTestInfoJson` for it to make more sense.

Reply:
Same thing for `resBenchmarkTestJson`.